public/locales/es/account.json

```diff
@@ -89,11 +89,10 @@
     "google": "Inicia sesión con Google",
     "hide_password": "Ocultar contraseña",
     "not_registered": "¿Nuevo/a en {{siteName}}?",
-    "ok": "Ok",
     "password": "Contraseña",
     "sign_in": "Iniciar sesión",
     "sign_up": "Registrarse",
-    "simultaneous_logins": "Se ha cerrado la sesión porque se ha alcanzado el límite de inicios de sesión simultáneos",
+    "simultaneous_logins": "Se ha alcanzado el número máximo de sesiones activas para esta cuenta. Iniciar sesión aquí lo cerrará en otra sesión o dispositivo.",
```

Comment:
We should better modify Spanish translations in a separate PR (as agreed).

Reply:
This is machine translations.It wil have separate PR for the final translations.